userspace/libsinsp/sinsp.h

```diff
@@ -144,6 +144,18 @@ class filter_check_info
 	uint32_t m_flags;
 };
 
+/*!
+  \brief Parameters to configure the download behavior when connected to an
+  orchestrator like Kubernetes or mesos.
+*/
+class metadata_download_params
+{
+public:
+	uint32_t m_data_max_mb = K8S_DATA_MAX_MB;
```

Comment:
Given how this is used below (as a byte counter), it should be `m_data_max_bytes` and `K8S_DATA_MAX_BYTES`

Reply:
Thank you. This should be addressed in efa93793dfa05e1ae26562ec57847bf7ec700525.